.github/workflows/migration-tests.yml

```diff
@@ -265,6 +265,8 @@ jobs:
           terraform_wrapper: false    
       - name: Migration Tests
         env:
+          MONGODB_ATLAS_ENABLE_PREVIEW: ${{ vars.MONGODB_ATLAS_ENABLE_PREVIEW }}
+          # can be removed when MONGODB_ATLAS_LAST_VERSION supports the new MONGODB_ATLAS_ENABLE_PREVIEW flag
           MONGODB_ATLAS_ENABLE_BETA: ${{ vars.MONGODB_ATLAS_ENABLE_BETA }}
```

Comment:
can we change to MONGODB_ATLAS_ENABLE_BETA: ${{ vars. MONGODB_ATLAS_ENABLE_PREVIEW }}

so we only have the GH var for preview?

Reply:
in fact not sure why do we have those vars, can we just do:
  MONGODB_ATLAS_ENABLE_PREVIEW: true
 MONGODB_ATLAS_ENABLE_BETA: true

and similarly in the other places?

i don't see values in having those cc @AgustinBettati

Reply:
done in 7597c2c ✅